packages/contracts/src/equipment/LibSpawnEquipmentSlots.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import { getUniqueEntity } from "@latticexyz/world/src/modules/uniqueentity/getUniqueEntity.sol";
+
+import { EqptBase, EqptBaseTableId } from "../codegen/Tables.sol";
+import { FromEqptBase, FromEqptBaseTableId } from "../codegen/Tables.sol";
+import { SlotAllowedBases, SlotAllowedBasesTableId } from "../codegen/Tables.sol";
+import { SlotEquipment, SlotEquipmentTableId } from "../codegen/Tables.sol";
+
+import { Name, NameTableId } from "../codegen/Tables.sol";
+import { OwnedBy, OwnedByTableId } from "../codegen/Tables.sol";
+
+library LibSpawnEquipmentSlots {
+  function spawnEquipmentSlots(bytes32 ownerEntity) internal {
+    bytes32 WEAPON = _newSlotEquipment(ownerEntity, "WEAPON");
+    bytes32 SHIELD = _newSlotEquipment(ownerEntity, "SHIELD");
+    bytes32 HAT = _newSlotEquipment(ownerEntity, "HAT");
+    bytes32 CLOTHING = _newSlotEquipment(ownerEntity, "CLOTHING");
+    bytes32 GLOVES = _newSlotEquipment(ownerEntity, "GLOVES");
+    bytes32 PANTS = _newSlotEquipment(ownerEntity, "PANTS");
+    bytes32 BOOTS = _newSlotEquipment(ownerEntity, "BOOTS");
+    bytes32 AMULET = _newSlotEquipment(ownerEntity, "AMULET");
+    bytes32 RING = _newSlotEquipment(ownerEntity, "RING");
```

Comment:
`LibInitEquipment` is nice, but you still duplicating it here which defeats the whole point